### ISSUE TITLE
[BUG] Error specifying input file schema for assignment tasks

### ISSUE THREAD

**Issue by PARanOiA1120** [nesl/AutoGrader#139]
Will get "records are not consistent to schema" error.

Action to trigger this bug:
- Pick a random assignment and click edit
- Add schema in the input file schema field, and submit changes
- Click the edit button of any task under this assignment

There should be something wrong with the _create_or_modify_assignment_task method in task.py (line 29)

**Comment by TimeString** [comment]
Does this problem constantly happen in different assignments?

**Comment by PARanOiA1120** [comment]
Yes. If you created task1 and then changed the schema, then we get error editing task1, since the files associated with task1 are no longer consistent with the updated schema.

**Comment by TimeString** [comment]
I see, yeah now I remember this issue. When I made the feature of allowing multiple input files, I was concerning that if an instructor has created some tasks but decided to change the shcema. I think we shouldn't let the end user to see the exception, but also we need to discuss the policy to deal with it.
